### PR TITLE
Add basic documentation

### PR DESCRIPTION
**Pull request by iocanel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,25 +1,108 @@
 # Quarkus Backstage
 
-[![Version](https://img.shields.io/maven-central/v/io.quarkiverse.backstage/quarkus-backstage?logo=apache-maven&style=flat-square)](https://search.maven.org/artifact/io.quarkiverse.backstage/quarkus-backstage)
+[![Version](https://img.shields.io/maven-central/v/io.quarkiverse.backstage/quarkus-backstage?logo=apache-maven&style=flat-square)](https://central.sonatype.com/artifact/io.quarkiverse.backstage/quarkus-backstage-parent)
 
-## Welcome to Quarkiverse!
+Generate Backstage Catalog Information as part of the Quarkus build or the Quarkus CLI.
```
Comment (cmoulliard) [inline README.md:5]: I propose that we rephrase the sentence to mention that we mainly create the Component YAML

"Generate the Backstage Catalog Information as the [component](https://backstage.io/docs/features/software-catalog/descriptor-format/) as part of the Quarkus build or the Quarkus CLI.

File: README.md [hunk 2]
```diff
@@ -1,25 +1,108 @@
...
 
-Congratulations and thank you for creating a new Quarkus extension project in Quarkiverse!
+## Features
 
-Feel free to replace this content with the proper description of your new project and necessary instructions how to use and contribute to it.
+- Generate the catalog-info.yaml for the Quarkus application
+- Command Line interface to install / uninstall and Component to Backsatge
```
Comment (cmoulliard) [inline README.md:10]: s/Backsatge/Backstage

File: README.md [hunk 3]
```diff
@@ -1,25 +1,108 @@
...
+- Orchestrate Quarkus extensions:
```
Comment (cmoulliard) [inline README.md:11]: What do you want to tell to the users using as word `orchestrate` ?

File: README.md [hunk 4]
```diff
@@ -1,25 +1,108 @@
...
+  - kubernetes
+  - helm
+  - argocd
 
-You can find the basic info, Quarkiverse policies and conventions in [the Quarkiverse wiki](https://github.com/quarkiverse/quarkiverse/wiki).
+## Requirements
 
-In case you are creating a Quarkus extension project for the first time, please follow [Building My First Extension](https://quarkus.io/guides/building-my-first-extension) guide.
+### For Catalog Info Generation
+To generate the catalog-info.yaml nothing special is required.
```
Comment (cmoulliard) [inline README.md:19]: This sentence is irrelevant if nothing is needed

File: README.md [hunk 5]
```diff
@@ -1,25 +1,108 @@
...
 
-Other useful articles related to Quarkus extension development can be found under the [Writing Extensions](https://quarkus.io/guides/#writing-extensions) guide category on the [Quarkus.io](https://quarkus.io) website.
+### For using the CLI / Client
+- A running Bacsktage installation with a known token
```
Comment (cmoulliard) [inline README.md:22]: A running Bacsktage installation with a known token =>
A running Backstage instance with a known token

Remark: Are you talking about this token https://backstage.io/docs/auth/service-to-service-auth#static-tokens ? If yes, please add the link to the backstage documentation

File: README.md (hunk 3 above)
Comment (iocanel) [reply README.md:11]: That it coordinates, configures extensions so that they work in harmony with each other and with `quarkus-backstage`.

Comment (cmoulliard) [reply README.md:11]: `Orchestrate` => `Operate closely with the following Quarkus extensions` maybe 
We should here mention that Backstage should have been configured to with their corresponding plugin !

File: README.md (hunk 4 above)
Comment (iocanel) [reply README.md:19]: I think that it is needed, cause things are required for other features of the extension (e.g. tokens etc). It should be clear that this is not the case when generating `catalog-info.yaml`

File: README.md (hunk 1 above)
Comment (iocanel) [reply README.md:5]: I don't agree with the proposal. It feels way too verbose.
We don't need to be more specific than that. Moreover, the catalog-info.yaml may contain other entities, too.

**Pull request merged**